test/unit/util/fsTest.ts

```diff
@@ -226,4 +226,83 @@ describe('util/fs', () => {
       expect(exists).to.be.false;
     });
   });
+
+  describe('areFilesEqual', () => {
+    afterEach(() => {
+      $$.SANDBOX.restore();
+    });
+
+    it('should return true if the files are the same', async () => {
+      $$.SANDBOX.stub(fs, 'readFile')
+        .onCall(0)
+        .resolves(
+          `{
+            "key": 12345,
+            "value": true,
+        }`
+        )
+        .onCall(1)
+        .resolves(
+          `{
+          "key": 12345,
+          "value": true,
+        }`
+        );
+      const exists = await fs.areFilesEqual('foo/bar.json', 'foo/bar2.json');
+      expect(exists).to.be.false;
```

Comment:
we should create files like below and use them in the 'areFilesEqual' call

Reply:
Agreed.